### PR TITLE
Configure capture modes

### PR DESCRIPTION
**Pull request by prsauer**
image

### CODE REVIEW

File: packages/desktop/components/Settings/RecordingSettings.tsx [hunk 1]
```diff
@@ -73,6 +83,36 @@ const recStates: Record<RecStatus | 'EngineNotStarted', { icon: JSX.Element; mes
   },
 };
 
+function PreviewVideoWindow({ key }: { key: string }) {
```
Comment (atinylittleshell) [inline packages/desktop/components/Settings/RecordingSettings.tsx:86]: hmm.. would it be possible for window resizing to affect the rect?

File: packages/desktop/components/Settings/RecordingSettings.tsx [hunk 2]
```diff
@@ -212,44 +252,107 @@ const RecordingSettings = () => {
...
+                          window.wowarenalogs.obs?.setConfig?.('obsCaptureMode', o.key);
+                        }}
+                      />
+                      <span className="label-text">{o.name}</span>
+                    </label>
+                  </div>
+                );
+              })}
+            </div>
+            {configStore?.obsCaptureMode === 'monitor_capture' && (
```
Comment (atinylittleshell) [inline packages/desktop/components/Settings/RecordingSettings.tsx:278]: what's the default value of this?

**Review by atinylittleshell — APPROVED** [review]

File: packages/desktop/components/Settings/RecordingSettings.tsx (hunk 1 above)
Comment (prsauer) [reply packages/desktop/components/Settings/RecordingSettings.tsx:86]: atm the div is statically sized with css

File: packages/desktop/components/Settings/RecordingSettings.tsx (hunk 2 above)
Comment (prsauer) [reply packages/desktop/components/Settings/RecordingSettings.tsx:278]: game_capture

**Pull request merged**